core/ir/opnd_shared.c

```diff
@@ -432,6 +461,19 @@ opnd_get_immed_float(opnd_t opnd)
     return opnd.value.immed_float;
 }
 
+#ifndef WINDOWS
+/* Type double currently not included for Windows because sizeof(opnd_t) does
+ * not equal EXPECTED_SIZEOF_OPND, triggering the ASSERT in d_r_arch_init().
```

Comment:
Always point at an issue with a TODO or XXX comment for unfinished/non-permanent situations.  Could put under #4488.

Reply:
I see you do have #4488 in one spot.  I think it's best if it's in each of these comments: there are a number of them.

Reply:
Done.